main-app/app/scripts/controllers.js

```diff
@@ -1,36 +1,52 @@
 (function() {
     'use strict';
     angular.module('Tombola.NoughtsAndCrosses')
-        .controller('MainController', function ($scope, $location, $http) {
+        .controller('MainController', function ($scope, $location, $http, $timeout) {
```

Comment:
We need to break this down in the next iteration of the code - along the sort of lines we discussed yesterday.
I'll review this stuff properly when it is separated.